packages/contracts/mud.config.ts

```diff
@@ -1,199 +1,248 @@
 import { mudConfig } from "@latticexyz/world/register";
 
 export default mudConfig({
```

Comment:
Could you please format the mud.config.ts file in its original way, as it's hard to see the differences now.